share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx

```diff
@@ -98,19 +117,18 @@
         <StaffType group="pitched">
           <name>stdNormal</name>
           </StaffType>
-        <defaultConcertClef>G8vb</defaultConcertClef>
-        <defaultTransposingClef>F</defaultTransposingClef>
+        <defaultClef>F</defaultClef>
```

Comment:
Why this change?

Reply:
In the old template both voice, female and male, used the same "instrument" <code>voice</code> and adapted the clef for male later. In the new template other instruments are used, <code>Women</code> for S/A and <code>Men</code> for T/B. Now the get automatically the correct clef.

Reply:
The nice part of the above was the easy switch between F and G8vb clef for the concert pitch button. Now that is lost

Reply:
Esp. the Tenors tend to hate the F Clef...

Reply:
Not the one I now (me :-))

Reply:
in our choir even one of the basses hates it (being a violinist too)

Reply:
Updated. Now all three SATB Closed templates are consistent.

Reply:
Thanks!